frontend/dockerfile/dockerfile_test.go

```diff
@@ -6310,6 +6311,61 @@ COPY --from=build out .
 	require.Equal(t, "freebsd", string(dt))
 }
 
+func testPlatformArgsExplicitContext(t *testing.T, sb integration.Sandbox) {
```

Comment:
I'm not so sure about correctness of this one and didn't have time to run this through. I ran unit tests though and they feel to be sufficient for the change